services/apps/alcs/src/providers/typeorm/migrations/1570975355875-initial_migration_v2.ts

```diff
@@ -0,0 +1,549 @@
+      `ALTER TABLE "alcs"."covenant" ADD CONSTRAINT "FK_0fa742d800bc0e0b3f2451e0f0b" FOREIGN KEY ("local_government_uuid") REFERENCES "alcs"."application_local_government"("uuid") ON DELETE NO ACTION ON UPDATE NO ACTION`,
+    );
+    await queryRunner.query(
+      `ALTER TABLE "alcs"."covenant" ADD CONSTRAINT "FK_a62eb808822aabedb56e1c9d928" FOREIGN KEY ("region_code") REFERENCES "alcs"."application_region"("code") ON DELETE NO ACTION ON UPDATE NO ACTION`,
+    );
+    await queryRunner.query(
+      `ALTER TABLE "alcs"."notification" ADD CONSTRAINT "FK_fc0e9a26b0a8f7c76658ca1c6ca" FOREIGN KEY ("actor_uuid") REFERENCES "alcs"."user"("uuid") ON DELETE NO ACTION ON UPDATE NO ACTION`,
+    );
+    await queryRunner.query(
+      `ALTER TABLE "alcs"."notification" ADD CONSTRAINT "FK_b776eec36c2a6b6879c14241e91" FOREIGN KEY ("receiver_uuid") REFERENCES "alcs"."user"("uuid") ON DELETE NO ACTION ON UPDATE NO ACTION`,
+    );
+    await queryRunner.query(
+      `ALTER TABLE "alcs"."planning_review" ADD CONSTRAINT "FK_735dcdd4fa909a60d0fa1828f24" FOREIGN KEY ("card_uuid") REFERENCES "alcs"."card"("uuid") ON DELETE NO ACTION ON UPDATE NO ACTION`,
+    );
+    await queryRunner.query(
+      `ALTER TABLE "alcs"."planning_review" ADD CONSTRAINT "FK_5a57c8d407eb6132ed39cb8fe6f" FOREIGN KEY ("local_government_uuid") REFERENCES "alcs"."application_local_government"("uuid") ON DELETE NO ACTION ON UPDATE NO ACTION`,
+    );
+    await queryRunner.query(
+      `ALTER TABLE "alcs"."planning_review" ADD CONSTRAINT "FK_409473aef1f92b6675e3f7c00ad" FOREIGN KEY ("region_code") REFERENCES "alcs"."application_region"("code") ON DELETE NO ACTION ON UPDATE NO ACTION`,
+    );
+    await queryRunner.query(
+      `ALTER TABLE "alcs"."modified_decisions" ADD CONSTRAINT "FK_733d6d17b98e6b6d3cd83359101" FOREIGN KEY ("application_modification_uuid") REFERENCES "alcs"."application_modification"("uuid") ON DELETE CASCADE ON UPDATE CASCADE`,
+    );
+    await queryRunner.query(
+      `ALTER TABLE "alcs"."modified_decisions" ADD CONSTRAINT "FK_0e1b988c373de06ec277bca3692" FOREIGN KEY ("application_decision_uuid") REFERENCES "alcs"."application_decision"("uuid") ON DELETE NO ACTION ON UPDATE NO ACTION`,
+    );
+    await queryRunner.query(
+      `ALTER TABLE "alcs"."reconsidered_decisions" ADD CONSTRAINT "FK_d7e09c1b2ce86005b7d0465215f" FOREIGN KEY ("application_reconsideration_uuid") REFERENCES "alcs"."application_reconsideration"("uuid") ON DELETE CASCADE ON UPDATE CASCADE`,
+    );
+    await queryRunner.query(
+      `ALTER TABLE "alcs"."reconsidered_decisions" ADD CONSTRAINT "FK_09a58410db3f332490181279771" FOREIGN KEY ("application_decision_uuid") REFERENCES "alcs"."application_decision"("uuid") ON DELETE NO ACTION ON UPDATE NO ACTION`,
+    );
+  }
+
+  public async down(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `ALTER TABLE "alcs"."reconsidered_decisions" DROP CONSTRAINT "FK_09a58410db3f332490181279771"`,
```

Comment:
Not that we would 'down' the initial migration, but is there going to be an issue with this uniquely named constraints not matching?

Reply:
The constraint names are generated by hashing the options of the relationship, this is how TypeORM knows if you change something on an existing relationship as the key names won't match. The good news is that theese IDs match what already exist in production as I haven't changed any relationship settings.